src/Storefront/Resources/views/storefront/element/cms-element-form/form-types/contact-form.html.twig

```diff
@@ -55,7 +55,7 @@
                 %}
             {% endblock %}
 
-            {% block cms_form_contact_input_phome %}
+            {% block cms_form_contact_input_phone %}
```

Comment:
Thx for your contribution @AEYCEN 🎉 

Like @mitelg already mentioned, we need to deprecate the old block name and cannot change it right away. You can do it like this and offer the new block already as an outer block. And in the next major the inner block with the typo can be removed. Please also mention the block as deprecated in the changelog.

```diff
+{% block cms_form_contact_input_phone %}
+    {# @deprecated tag:v6.6.0 - Block will be renamed because of a typo. Use `cms_form_contact_input_phone` instead. #}
    {% block cms_form_contact_input_phome %}
    {% endblock %}
+{% endblock %}
```

You can also find [examples in shopware](https://github.com/shopware/shopware/blob/trunk/src/Storefront/Resources/views/storefront/component/line-item/type/product.html.twig#L77) where we did it similarly to adjust block names.